build.sh

```diff
@@ -36,8 +36,12 @@ while (( "$#" )); do
     runtests=false
   elif [[ "$1" == "--diff" ]]
   then
-    apis+=($(git diff main --name-only | grep -e 'apis/.*/' | cut -d/ -f 2 | uniq))
-    diff=true
+    if git diff main --name-only | grep -q "Directory.Packages.prop"; then
```

Comment:
Possibly add a comment (or modify the message) to make it clear that the regex can still filter this?

(My understanding is that build-pr.yml will still effectively split this into three build jobs, each of which will only build a subset, yes?)

Reply:
I've added the other "top level dependencies" file and changed the message. And yes, to clarify my las comment that was confusing, on build.sh the API list will still be filtered by regex. It is build-pr action that takes care to pass regexes that will end up building everything that's on the original API list. See if the comment is good enough now.